examples/local_sgd/resnet/train_with_fleet.py

```diff
@@ -89,30 +88,56 @@ def set_paddle_flags(flags):
 
 add_arg('use_gpu',                      bool,   True,                 "Whether to use GPU or not.")
 add_arg('nccl_comm_num',                int,    1,                    "nccl comm num")
-add_arg('num_iteration_per_drop_scope', int,    10,                   "Ihe iteration intervals to clean up temporary variables.")
+add_arg('num_iteration_per_drop_scope', int,    30,                   "Ihe iteration intervals to clean up temporary variables.")
 
 add_arg('use_local_sgd',                bool,   True,                 "Whether to use LocalSGD argorithmn.")
 add_arg('local_sgd_steps',              int,    2,                    "The step number for local training before synchronizing parameters.")
+add_arg('local_sgd_is_warm_steps',      int,    30,                   "The warmup step of number for local sgd.")
+add_arg('lsgd_warmup_strategy',         int,    1,                    "Select strategy to warmup the lsgd,1:exp,2:const,3:linear")
+add_arg('isTest',                       bool,   False,                "Whether to test on every epoch")
```

Comment:
why the format of this argument is different to others, you should prefer `is_test`.

Reply:
Done